actioner/README.md

```diff
@@ -0,0 +1,33 @@
+# Actioner
+
+## Overview
+
+Alternative way to connect your Datadog account with Actioner.
+
+## Setup
+
+Follow the steps below to establish connection between Actioner and Datadog accounts.
+
+1. Log in to your Actioner account.
+2. Go to profile settings page from the bottom-left corner. Switch to my connections tab.
+3. Click on the "Add connection", select Datadog connection. Proceed with the "Connect"
```

Comment:
The Oauth process should be kicked off from Datadog. Please update the the instructions to start at Datadog and the steps the customer will take